src/components/ChatLog.jsx

```diff
@@ -0,0 +1,39 @@
+import './ChatLog.css';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+const ChatLog = (props) => {
+  const chatComponents = props.entries.map((message, index) => {
+    return (
+      <div key={index}>
+        <ChatEntry
+          id={message.id}
+          sender={message.sender}
+          body={message.body}
+          timeStamp={message.timeStamp}
+          liked={message.liked}
+          toggleLikes={props.onChatEntryToggleLikes}
+          local={props.local}
+          localColor={props.localColor}
+          remoteColor={props.remoteColor}
+        ></ChatEntry>
+      </div>
+    );
+  });
+  return <>{chatComponents}</>;
```

Comment:
Nice job capturing the return value from invoking `map` in the variable `chatComponents` and using that variable here. This is typical of what you'd see in industry!